src/tapedfunction.jl

```diff
@@ -61,11 +53,10 @@ mutable struct TapedFunction{F} <: Taped
         tf.func = f
         tf.tape = RawTape()
         tf.counter = 1
-        tf.block_map = Dict{Int, Int}()
 
         if init # init
             tf.arity = length(args)
-            ir = IRTools.@code_ir tf.func(args...)
+            ir = code_inferred(tf.func, typeof.(args)...)
```

Comment:
Simpler and without potentially slow broadcasting and splatting:
```suggestion
            ir = code_inferred(tf.func, typeof(args))
```
(One can provide the type of the arguments also as a `Tuple{T1,T2,...}`).